p2p_wallet/AppDelegate.swift

```diff
@@ -83,4 +88,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     ) {
         debugPrint("Failed to register: \(error)")
     }
+
+    func application(
+        _: UIApplication,
+        didReceiveRemoteNotification _: [AnyHashable: Any],
+        fetchCompletionHandler _: @escaping (UIBackgroundFetchResult) -> Void
+    ) {
+        moveToHistory()
```

Comment:
Let's use `RxAppState` to observe this notification in TabBarVC (or Scene that we want to make action), keep `AppDelegate` slim